sdk/appconfiguration/app-configuration/src/appConfigurationClient.ts

```diff
@@ -279,7 +274,7 @@ export class AppConfigurationClient {
    *
    * Example code:
    * ```ts
-   * const allSettingsWithLabel = client.listConfigurationSettings({ labelFilter: "MyLabel" });
```

Comment:
I think you wanted this to be `labelFilter`.

Reply:
Gah, I can't believe I did that. The work I did in this PR was based off another one that was based off an older version....I've gone through and checked and the rest of the changes should be proper. Thanks for catching this.